src/dosbox.cpp

```diff
@@ -504,34 +505,36 @@ void DOSBOX_Init()
 	pstring = secprop->Add_string("vmemsize", only_at_start, "auto");
 	pstring->Set_values(vmemsize_choices);
 	pstring->Set_help("Video memory in MiB (1-8) or KiB (256 to 8192). 'auto' uses the default per\n"
```

Comment:
Again, spelling mistake: MB, KB, not KiB or MiB.

Reply:
Good catch, @FeralChild64. 
This one is sure frustrating - because even though metric and IEC define 'MB' in generate to mean the base-10 values and 'MiB' for base-2, Microsoft unilaterally used 'MB' specifically to refer to memory in base-2.

Given we're in Microsoft's land of DOS and dealing with a memory size, I think we should go w/ MB here, too.

Reply:
This nuance was the trick that HDD manufactures quickly latched onto, given they weren't beholden to Microsoft terms.

So they could also use 'MB' and 'GB' units, and legally deliver less storage user the metric and IEC base-10 units :man_facepalming: :joy:

Reply:
(Ref: https://en.wikipedia.org/wiki/Megabyte)

Reply:
I totally forgot about this idea, I must admit. It clearly didn't catch up at all: not only Microsoft still uses KB, MB, GB - check the specification of DDR RAM you can buy today (can you find 32 GiB RAM anywhere?), how NVidia and AMD describe the amount of memory on their cards, how Intel describes the amount of cache on their CPUs, etc. Even Wikipedia uses MB (not MiB): https://en.wikipedia.org/wiki/Windows_3.1x

I haven't actually seen these "MiB" units anywhere since probably like 10 years or so.

Reply:
I am used to seeing it in tenders, particularly for storage. To prevent the storage vendor from using base10 for capacity or throughput. Storage vendors have a long history with using base10 MB/GB/TB/PB numbers, instead of base2 as the rest of the industry tends to use.

Reply:
Okay, changing all of them to KB and MB then, as we're not storage vendors, and this is not a tender 😝